files/usr/lib/jeos-firstboot

```diff
@@ -352,16 +352,14 @@ for p in /sys/class/net/* ; do
         eval `wicked test dhcp4 "$d" 2>/dev/null | grep -E "^IPADDR=|^GATEWAYS="`
         ip link set down "$d" # set link down after probe once done
 
-        test -n "$IPADDR"   && candidates+=("$d")  || continue # ok, track it
-        test -n "$GATEWAYS" && candidates=("$d")  || continue # just use it
-        break
+	# Collect the interfaces that provides IPADDR and GATEWAYS
+	[ -n "$IPADDR" -a -n "$GATEWAYS" ] && candidates+=("$d")
 done
 
 for d in ${candidates[@]} ; do
         rm -f "/etc/sysconfig/network/ifcfg-$d" || exit 1
```

Comment:
Can be merged into the loop above now and `candidates` removed

Reply:
Sure. Done